interface-elements-for-desktop/articles/rich-text-editor.md

```diff
@@ -84,6 +84,11 @@ This section describes the capabilities provided by the **Rich Text Editor**.
 
```

Comment:
Since I cannot edit the lines above, please make the following changes:

Line 8: `This section describes the capabilities provided by the **Rich Text Editor**.`
I'd like to rewrite this without using "provided". Can we say the following?

`This section describes the **Rich Text Editor** capabilities.`

Line 27: `* [Divide a Documents into Sections](rich-text-editor/document-layout-and-page-setup/divide-a-document-into-sections.md)`

Please change the topic title to "Divide a Document into Sections".